src/generated/meshtastic.rs

```diff
@@ -5111,7 +5111,11 @@ pub struct MeshPacket {
     #[prost(fixed32, tag = "1")]
     pub from: u32,
     ///
-    /// The (immediate) destination for this packet
+    /// The (immediate) destination for this packet.
```

Comment:
This is not the right place for it. As the name suggests, this is a generated file. This will get overwritten when we update protobuf and regenerate this file.

Please post a PR to the protobuf repository:
https://github.com/meshtastic/protobufs/blob/7654db2e2d1834aebde40090a9b74162ad1048ae/meshtastic/mesh.proto#L1373